typescript/react/portability/i18next/jsx-not-internationalized.yaml

```diff
@@ -0,0 +1,21 @@
+rules:
+- id: jsx-not-internationalized
+  patterns:
+  - pattern: <$ELEMENT>$MESSAGE</$ELEMENT>
+  - pattern-not: <$ELEMENT>t('$KEY', ...)</$ELEMENT>
+  message: 'JSX element not internationalized: ''$MESSAGE'''
+  languages:
+  - ts
+  - js
+  severity: WARNING
+  metadata:
+    category: portability
+    technology:
+    - react
+    - mui
+    - i18next
+    source-rule-url: https://www.soluvas.com/
```

Comment:
`source-rule-url` is used for linking to a ported rule - is there a more specific page of soluvas.com describing the specific approach used in this rule?